solidity/test/TBTCSystemTest.js

```diff
@@ -33,14 +33,15 @@ describe("TBTCSystem", async function() {
     let openKeepFee
     const tdtOwner = accounts[1]
     const keepOwner = accounts[2]
+    const tokenStakeDuration = 123
```

Comment:
can we rename this to make it clear this is a lock?
maybe `stakeLockDuration`, like before

Reply:
Renamed it to `maxSecuredLifetime` as per https://github.com/keep-network/tbtc/pull/566#discussion_r406559607